lib/index.js

```diff
@@ -16,8 +16,13 @@ function register (server, options, next) {
   var addOrgName = function (orgName, cb) {
     return query(queries.addOrgName(orgName), pool, cb);
   };
-  var getActiveOrgs = function (cb) {
-    return query(queries.getActiveOrgs, pool, cb);
+  var orgsGetByTag = function (active, filter, cb) {
+    return query(queries.orgsGetByTag(active, filter), pool,
+      function (err, rows) {
+        var res = !err ? formatters.orgsGetByTag(rows) : err;
```

Comment:
If there is an error then res = err and then you return in the callback cb (err, res) where the response is the error. I think it's better to return null, maybe something like ```var res = !err ? formatters.orgsGetByTag(rows) : null```

or 

```
if (err) {
  return cb(err, null);
} 
// format the data here
return cb(null, response);
```